vllm/model_executor/model_loader/weight_utils.py

```diff
@@ -262,6 +266,11 @@ def download_weights_from_hf(
             revision=revision,
             local_files_only=huggingface_hub.constants.HF_HUB_OFFLINE,
         )
+        end_time = time.time()
```

Comment:
Better to use perf.counter() or monotonic() instead.

Reply:
@comaniac Thanks for the good catch!
Force-push to https://github.com/vllm-project/vllm/commit/23f62ea2caeea576dc0579ffb7e7c86c7400e77f used `perf_counter()` instead.